profiles/base-crafting.yaml

```diff
@@ -252,6 +252,9 @@ remedies:
     stock-number: 1
     stock-name: water
     stock-volume: 10
+    stock-number_a: 2
+    stock-name_a: alcohol
+    stock-volume_a: 10
     pattern-book: remed
```

Comment:
We should stick with dashes our underscores, at the very least in the same variable.